readme.txt

```diff
@@ -1,9 +1,9 @@
 === Maintenance Mode ===
```

Comment:
Keep changelog updates separate from the PR - they can be put together just before a release is made.

Also see https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/ - in a separate PR you could scrap the `readme.txt` file completely and move it all over to `readme.md`

Reply:
That was new to me and I moved the content from `readme.txt` to `readme.md` and removed `readme.txt`.